docker-for-mac/index.md

```diff
@@ -41,14 +41,20 @@ menu bar and configure the runtime options described below.
 
 On the **General** tab, you can configure when to start and update Docker:
 
+- **Automatically check for updates**: By default, Docker Desktop is configured to automatically check for newer versions. [Upgrade your existing organization to a Team plan](https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade) to disable the automatic check for updates. If you have installed Docker Desktop as part of an organization, you may not be able to update Docker Desktop. Consult your organization administrator to update Docker Desktop to a newer version.
```

Comment:
This paragraph seems in the wrong order. I think we should say:

```suggestion
- **Automatically check for updates**: By default, Docker Desktop is configured to check for newer versions automatically. If you have installed Docker Desktop as part of an organization, you may not be able to update Docker Desktop yourself. In that case, [upgrade your existing organization to a Team plan](https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade) and clear this checkbox to disable the automatic check for updates.
```

(ditto on Windows, of course).

Reply:
Done